### ISSUE TITLE
Multiple candidates upon sass @import

### ISSUE THREAD

**Issue by Paveltarno** [mdbootstrap/mdb-ui-kit#760]
First of all, thanks a lot for this beautiful theme [=

A small issue:
I'm referencing the SASS version main file from my own .scss stylesheet:

```
// Main manifest file
@import "bootstrap";
@import "font-awesome";
@import "bootstrap-material-design";
```

Problem is, there are two files with that name, one being a partial called `_bootstrap-material-design.scss` and the other `bootstrap-material-design.scss`. When sass (i'm using libsass) tries to import the main file, it doesn't know which one to load:

```
Error: It's not clear which file to import for '@import "bootstrap-material-design"'.
       Candidates:
         _bootstrap-material-design.scss
         bootstrap-material-design.scss
       Please delete or rename all but one of these files.
```

This is not an issue with LESS where the `@import` directive is more explicit:

_bootstrap-material-design.less:_
`
@import '_bootstrap-material-design.less'
`

I thought about changing the `_bootstrap-material-design.less` file name to something like `_core.less` then, when transpiled to SASS will solve the problem.  
What do think? Should I submit a PR?

**Comment by rosskevin** [comment]
_core is good.  PR and I'll include it today.  Sooner the better

**Comment by rosskevin** [comment]
I'll get this now, I'm in it for a few fixes.

**Comment by rosskevin** [comment]
Available as bower 0.5.3
